docs/user/openstack/README.md

```diff
@@ -69,6 +69,25 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
 
 ## OpenStack Requirements
 
+Before starting to install OpenShift, ensure following OpenStack services are
```

Comment:
The OpenShift installation on OpenStack platform relies on a number of core services being available.

Reply:
ack